numpy/core/src/multiarray/conversion_utils.c

```diff
@@ -540,6 +540,15 @@ PyArray_OrderConverter(PyObject *object, NPY_ORDER *val)
         return ret;
     }
     else if (!PyBytes_Check(object) || PyBytes_GET_SIZE(object) < 1) {
+        int ret = DEPRECATE("Non-string object detected for " \
```

Comment:
Yeah, this is right. Can you add a comment /* 2015-12-14, 1.11 */ above both? I think the `\` might be unnecessary here, since it is all in brackets? Or was that different in C, don't remember?

Other nitpick, I think I prefer order in lower case maybe with backticks, but frankly I doubt we have a consistent style, so not unless someone else complains soon. I am not sure we should name ORDER explicitly, since in principle the argument could be called differently. Don't have a much nicer idea though, maybe someone else has ;).